db/migrate/20140522185700_change_templatekind_to_bootdisk.rb

```diff
@@ -1,12 +1,16 @@
 class ChangeTemplatekindToBootdisk < ActiveRecord::Migration
+  class FakeConfigTemplate < ActiveRecord::Base
+    self.table_name = 'config_templates'
```

Comment:
I'd bet this fails tests for the same reason mentioned in https://github.com/theforeman/foreman_bootdisk/pull/13/files#r32195063.

It's a bit tricky since the ordering of DB migrations from plugins is going to depend on whether they're run together or separately.  Both are valid.